cloudify_cli/config/parser_config.py

```diff
@@ -37,12 +37,24 @@ def blueprint_id_argument():
     }
 
 
+def snapshot_id_argument(hlp = 'The id of the snapshot'):
```

Comment:
can we remove this default value? It is generated elsewhere. Also, it isn't consistent with the rest of the argument definitions.